README.md

```diff
@@ -26,6 +27,25 @@ the function and run automatically when you commit through a GitHub action.
 What is the worst-case big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
 
+This implementation of Depth First Seach has complexity $\Theta (V^2)$.
+There is a while loop that iterates over every V in the worst case and inside the
+while loop there is a for loop that iterates V times as well if a node is 
+connected to each other node. There is also 2 for loop outside the while loop
```

Comment:
...and what if the graph isn't completely connected?

Reply:
Sorry this was supposed to be in the worst case. In the case where the graph isnt completely connected it would not run this many times. I have updated the readme now.